src/test/java/com/karankumar/bookproject/backend/repository/BookRepositoryTest.java

```diff
@@ -87,7 +90,7 @@ void successfullyDeleteSingleAuthoredBook() {
         bookRepository.delete(book);
 
         // then
-        assertThat(bookRepository.findAll()).isEmpty();
+        assertThat(bookRepository.findAll().size()).isEqualTo(2);
```

Comment:
Could you please change this test to make it clear what the test is doing without needing to look at the other code? For example, I think it would be clearer here if we create an `Author` and `Book` in this method to show that the author only has one book